llvm/test/MachineVerifier/test_g_multiple_errors.mir

```diff
@@ -0,0 +1,22 @@
+# RUN: not --crash llc -mtriple=aarch64 -o /dev/null -run-pass=none -verify-machineinstrs %s 2>&1 | FileCheck %s --implicit-check-not="Bad machine code"
+# REQUIRES: aarch64-registered-target
+
+# Check that errors are reported from only one function.
```

Comment:
This behavior is really annoying if it's fixable

Reply:
Even before this patch, `AbortOnError` defaults to true which means we will abort after reporting the first error.

https://github.com/llvm/llvm-project/blob/a075e785b8f4e2323ce89b742185386314909f21/llvm/lib/CodeGen/MachineVerifier.cpp#L393-L394

The problem I encountered is that if the report is super long, it's possible that the verifier would find another error and start print it before the program aborts. The output would be unreadable since two separate errors were printing at the same time.

This patch aims to fix that scenario by ensuring a single error gets reported at a time. Since we abort after the first error, this test checks that we only print errors from one function.

Reply:
I updated the wording to be a bit more clear.